magpylib/_lib/classes/currents.py

```diff
@@ -205,3 +205,7 @@ def getB(self, pos):  # Particular Line current B field calculation. Check RCS f
         rotatedPos = rotateToCS(pos, self)
         return getBField(Bfield_CurrentLine(rotatedPos, self.vertices, self.current),  # The B field
                          self)
+
+
+def __repr__(self):
+    return f"current:  {self.current}, vertices: {self.vertices}, position:  {self.position}, angle:  {self.angle}, axis:  {self.axis}"
```

Comment:
Let's remove the vertices from the output for this one. It can get really nasty if it comes out on the console (think of a very long spiral with tons of turns).


These are missing the units for current (A), position (mm) and angle (degrees), and some line breaks 
like the request suggested:

![image](https://user-images.githubusercontent.com/7332704/60668203-ad700600-9e6b-11e9-8b38-de885e05d849.png)